src/speak/rest.rs

```diff
@@ -146,7 +146,7 @@ mod tests {
 
     #[test]
     fn listen_url() {
-        let dg = Deepgram::new("token");
+        let dg = Deepgram::new("token").unwrap();
```

Comment:
This is a concern called out in the tracking doc on item #8

Reply:
Yep.  I updated Deepgram::new() to return a `Result<Deepgram>`, so the caller can decide whether to panic.  Right here, tests `.unwrap()` to signal failure, so this is an appropriate use.